fragdenstaat_de/fds_blog/templates/fds_blog/_article_detail.html

```diff
@@ -169,28 +165,35 @@ <h3 class="small text-secondary">
             {% endwith %}
         {% endblock article_languages %}
         {% block article_suggestions %}
-            {% with next_read=article|get_next_read %}
-                {% if next_read %}
+            {% with similar_articles=article.get_similar_articles|slice:":3" %}
+                {% if similar_articles %}
                     <section>
-                        <div class="container-lg">{% include "fds_blog/includes/blog_item_list.html" with article=next_read %}</div>
+                        <div class="container-lg">
+                            <h3 class="small text-secondary">{% trans "Similar articles" %}</h3>
+                            <div class="row g-3 g-lg-4 row-cols-1 row-cols-md-2 row-cols-lg-3">
+                                {% for article in similar_articles %}
+                                    {% include "fds_blog/includes/blog_item_short.html" with article=article %}
+                                {% endfor %}
+                            </div>
+                        </div>
                     </section>
                 {% endif %}
             {% endwith %}
-        {% endblock article_suggestions %}
-        {% block article_previous %}
             {% if previous_articles %}
                 <section>
                     <div class="container-lg">
                         <h3 class="small text-secondary">
                             {% blocktranslate count counter=previous_articles|length %}
-                            Previous article:
-                        {% plural %}
-                            Previous articles:
-                        {% endblocktranslate %}
+                                Previous article:
+                            {% plural %}
+                                Previous articles:
+                            {% endblocktranslate %}
                         </h3>
-                        {% for related_article in previous_articles %}
-                            {% include "fds_blog/includes/blog_item_list.html" with article=related_article %}
-                        {% endfor %}
+                        <div class="row g-3 g-lg-4 row-cols-1 row-cols-md-2 row-cols-lg-3">
+                            {% for related_article in previous_articles %}
+                                {% include "fds_blog/includes/blog_item_short.html" with article=related_article %}
+                            {% endfor %}
+                        </div>
                     </div>
                 </section>
             {% endif %}
```

Comment:
Not sure if this is within the scope of this PR, but I find the heading "Previous article(s)" a bit misleading as this section doesn’t show chronologically previous articles but a curated list of related ones defined in the admin. But I'm not sure what the original intention was and how best to distinguish this section from "Similar articles" otherwise.

Reply:
Actually, previous articles should be chronologically before the current article. The feature is mostly used for series of articles (i.e. lawsuit article → update article → ruling article), where there is a semantic and timely connection.

I agree that the wording with related and similar articles is a little confusing.